DhsTranslator/DhsTranslator.spec

```diff
@@ -0,0 +1,74 @@
+%define _prefix __auto__
+%define gemopt opt
+%define name DhsTranslator
+%define version __auto__
+%define release __auto__
+%define repository gemini
+
+%define debug_package %{nil}
+
+Summary: %{name} Package
+Name: %{name}
+Version: %{version}
+Release: %{release}.%{dist}.%{repository}
+License: GPL
+## Source:%{name}-%{version}.tar.gz
+Group: Gemini
+Source0: %{name}-%{version}.tar.gz
+BuildRoot: /var/tmp/%{name}-%{version}-root
+BuildArch: %{arch}
+Prefix: %{_prefix}
+## You may specify dependencies here
+BuildRequires:axis2c-unofficial-devel%{?_isa} dhsClient-devel%{?_isa} drama-devel%{?_isa}
+Requires: axis2c-unofficial%{?_isa} drama%{?_isa} dhsClient%{?_isa}
```

Comment:
I may not remember this too well, will this require the `axis2c-unofficial` rpm from our yum repo?
Is it there?